sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleController.java

```diff
@@ -316,7 +320,7 @@ public void onDiscard() {
 			}
 		});
 
-		if (UserProvider.getCurrent().hasUserRight(UserRight.SAMPLE_DELETE)) {
+		if (!hideReferAndDeleteButton && UserProvider.getCurrent().hasUserRight(UserRight.SAMPLE_DELETE)) {
```

Comment:
would be better to have all the code below inside an if statement instead of adding the same condition in multiple places:

```
if (!hideReferAndDeleteButton) {
....
}
```

Also I would prefer to use a variable name that must not be negated in the condition

Reply:
This is supposed to be temporary, so I wanted to change as little as possible.
Also I would argue against a nested if statement, because it is easier to overlook in this rather large method with rather large sections.
Do you insist on this change?